spec/models/user_access_spec.rb

```diff
@@ -70,13 +70,14 @@
 
         it "returns the organizations an admin can perform actions on" do
           permission_matrix.each do |admin, action, current_resource, expected_resources|
-            admin.accesses.create(resource: current_resource)
+            admin.accesses.create!(resource: current_resource)
```

Comment:
All of these tests were silently re-testing the same cases for each user, because the second `create` would fail, so we wouldn't have the new user access, so we were just re-testing the first case for every user in the matrix.

Gotta use `create!` for creating models via AR in tests like this, always.

Reply:
great catch 👏🏼